services/web/server/src/simcore_service_webserver/projects/projects_handlers.py

```diff
@@ -39,6 +40,7 @@
     get_project_unavailable_services,
     project_uses_available_services,
 )
+from .. import sharing_networks_core
```

Comment:
hmm I think you will not get away with this one @pcrespov?

Reply:
Sorry I miss the point, am I supposed to expose it differently?